### ISSUE TITLE
HiC track regression? ReferenceError buffer not defined in 2.15.3?

### ISSUE THREAD

**Issue by fubar2** [GMOD/jbrowse-components#4583]
**Describe the bug**
2.15.0 works, but a simple test with a HiC track fails in 2.15.3 with
![image](https://github.com/user-attachments/assets/3b6c6bfa-981c-4512-8998-0dbf5f35c773)

**To Reproduce**
```
jbrowse create jb2.15.3 -t v2.15.3 
cd jb2.15.3/
jbrowse add-assembly --load symlink ../hg19.fa 
jbrowse add-track --load symlink ../GSE63525_HMEC_combined.hic 
cd ..
jbrowse create jb2.15.0 -t v2.15.0 
cd jb2.15.0/
jbrowse add-assembly --load symlink ../hg19.fa 
jbrowse add-track --load symlink ../GSE63525_HMEC_combined.hic 

```

**Expected behavior**
With 2.15.0:

![image](https://github.com/user-attachments/assets/52d2a102-0ae0-41fd-9f7f-f4b297a93131)

**Screenshots**

**Version:**
2.15.3 vs 2.15.0

**Comment by cmdcolin** [comment]
thanks so much for reporting this. I can confirm, there were some changes to the build to try to avoid the use of nodejs polyfills and this was a casualty....fix in #4584

**Comment by cmdcolin** [comment]
went ahead an cut a release to fix this in v2.15.4!

**Comment by fubar2** [comment]
Thanks @cmdcolin - that fixed it - thank you very much for the quick fix!
